src/99-appendices/10-file-collections.md

```diff
@@ -0,0 +1,53 @@
+# Appendix X: File collections
+
+To give context to the use of [entity-linked file collections](../02-common-principles.md#file-name-structure), applications concerned with this
+common principle are listed by descriptive tables under their respective modalities.
+
+Note that the tables in this appendix catalog applications where the use of
+a file collection is REQUIRED.
+
+Certain entitites interlink the files in a file collection through a metadata field.
+Unlike other common entities (for example `run`), they require an iteration over different
+values of the metadata fields they represent.
+Please keep the following list of linking entities up-to-date with the file collections
+included in this appendix:
+
+* Magnetic Resonance Imaging
+    * [`echo`](./09-entities.md#echo)
+    * `flip` (to be added)
+    * `inv`  (to be added)
+    * `mt`   (to be added)
+    * `part` (to be added)
```

Comment:
some of these have been added now

Reply:
I did that in https://github.com/bids-standard/bids-specification/pull/668/commits/7e799b5364ed581395a287316310b3397fa87266